pkgs/misc/tmux-plugins/tmux-thumbs/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, mkTmuxPlugin, fetchFromGitHub, thumbs }:
+mkTmuxPlugin rec {
+  pluginName = "tmux-thumbs";
+  version = "0.7.1";
+  rtpFilePath = "tmux-thumbs.tmux";
+  src = fetchFromGitHub {
+    owner = "fcsonline";
+    repo = pname;
```

```suggestion
    repo = pluginName;
```
Sorry, I forgot that this needed to change too

---

Np, I already fixed that ;)